File: 2018-01-PSITS-SultanKudarat/practice-2018-01-24/CSharp/TDDSample/GreeterTests.cs
using System;
using Xunit;

namespace TDDSample
{
    public class GreeterTests
    {
        [Fact]
        public void shouldReturnCorrectGreeting()
        {
            // arrange
            Greeter greeter = new Greeter();

            // act
            string greeting = greeter.Greet("Jboy");

            // assert
            Assert.Equal("Hello, Jboy!", greeting);
        }
        
        [Fact]
        public void shouldReturnCorrectGreeting_2()
        {
            // arrange
            Greeter greeter = new Greeter();

            // act
            string greeting = greeter.Greet("World");

            // assert
            Assert.Equal("Hello, World!", greeting);
        }
    }
}

File: 2018-01-PSITS-SultanKudarat/practice-2018-01-24/Java/src/GreeterTests.java
import org.junit.Assert;
import org.junit.Test;

public class GreeterTests {

    @Test
    public void shouldReturnCorrectMessage() {
        // arrange
        Greeter greeter = new Greeter();

        // act
        String greeting = greeter.greet("Jboy");

        // assert
        Assert.assertEquals("Hello, Jboy!", greeting);
    }
}
